build-plugin/action.yml

```diff
@@ -81,6 +85,27 @@ runs:
         printf "plugin-id=%s\n" "${{ steps.package-plugin.outputs.plugin-id }}"
         printf "plugin-version=%s\n" "${{ steps.package-plugin.outputs.plugin-version }}"
 
+    - name: Checkout code
+      if: ${{ inputs.use_changelog_generator == 'true' }}
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Generate Changelog
+      id: github-changelog-generator
+      if: ${{ inputs.use_changelog_generator == 'true' }}
+      uses: janheinrichmerker/action-github-changelog-generator@v2.4
```

Comment:
use a fixed hash instead of a moving tag. (replace `@v2.4` for the actual hash to the commit of this action)